### ISSUE TITLE
akku-king.net

### ISSUE THREAD

**Issue by wanhose** [wanhose/cookie-dialog-monster#682]
#### Browser
Firefox (128.0)
#### Reason
Popup showed up
#### URL
https://www.akku-king.net/akku-king-akku-ersetzt-jbl-gsp1029102a-li-polymer-6000mah-fuer-jbl-charge-3-2016-charge-3-2016-version/mp3-player-zubehoer/akku/jbl_20115069_34574
#### Version
7.0.4

**Comment by dreirund** [comment]
Cookie selection dialogue did show up.

Without interacting with it, on page reload it was gone.

---

![image](https://github.com/user-attachments/assets/99fe7f50-e0ca-4c39-8d9f-19cc18858fab)

---

**Comment by wanhose** [comment]
Hi @dreirund,

There are cases like this where the popup removal is delayed until the page internally loads. This is an expected behaviour in a few sites like this one so I'm going to close the issue. Feel free to add more information if needed.

Sorry for the inconveniences.

**Comment by dreirund** [comment]
I did install version 7.1.0 of this addon, updated the database, and opened the URL again (in the background).

I got the popup again. The addon's icon does not show any badge.

When I switch to it's tab and reload, the popup is removed.

I can reproduce it:

* Opening in the background &rarr; popup is there and stays.
* Opening/ reloading in the foreground &rarr; popup is removed.

---

![image](https://github.com/user-attachments/assets/2ea1eeeb-26fa-44b0-a562-d359d8db93cf)
